src/pages/en/guides/content.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Authoring Content
+description: "Astro is a perfect choice for your content-focused site: blogs, marketing sites, portfolios, and more! Author your content directly in your project, or connect your CMS of choice."
+i18nReady: false
+---
+Astro is a perfect choice for your content-focused site: blogs, marketing sites, portfolios, and more!
+
+Astro helps you author and present your content. You can write a blog post directly in Astro using Markdown/MDX, or fetch your content from a headless CMS. Astro lets you build a site around your content: you can add a layout to your pages, create an index of posts, and set up an RSS feed to allow readers to subscribe.
+
+## Writing Content
+
+In Astro, you can author your content in a variety of ways: 
+- In Markdown (`.md`) files, designed to make it easy to write rich text content.
+- In MDX (`.mdx`) files, which let you include components and dynamic expressions in your document.
+- Using a third-party content management system (CMS), then pulling that content into a `.astro` page.
+
```

Comment:
Agreed! My initial notes did include `.astro` files, and I think was before we were confidently documenting how to use `.html` files, so that's a good addition too. I like being comprehensive.